astropy/coordinates/tests/test_geodetic_representations.py

```diff
@@ -29,8 +29,45 @@ class CustomGeodetic(BaseGeodeticRepresentation):
     _equatorial_radius = 4000000.0 * u.m
 
 
+class IAUMARS2000GeodeticRepresentationEast180(BaseGeodeticRepresentation):
+    _equatorial_radius = 3396190.0 * u.m
+    _flattening = 0.5886007555512007 * u.percent
+
+
+class IAUMARS2000GeodeticRepresentationWest180(BaseGeodeticRepresentation):
```

Comment:
So, with my suggestion of a mixin class, this would be
`class ...(WestLongitude, BaseGeodecticRepresentation):`